lib/spree/adyen/form.rb

```diff
@@ -40,6 +55,47 @@ def url payment_method, endpoint
       Form.url(server, endpoint)
     end
 
+    def form_payment_methods_and_urls(response, order, payment_method)
```

Comment:
Could we pull some of these statements into other private methods? It looks like constructing an 'issuer' and 'payment method' would be a good candidate for refactor.